README.md

```diff
@@ -121,4 +121,4 @@ We highly encourage and welcome contributions to this package. Check out [this d
 # 🏪 Are there any resources available?
 - If you have questions or want to reach out for help, please refer to the [GitHub Issue](https://github.com/fivetran/dbt_twilio_source/issues/new/choose) section to find the right avenue of support for you.
 - If you would like to provide feedback to the dbt package team at Fivetran or would like to request a new dbt package, fill out our [Feedback Form](https://www.surveymonkey.com/r/DQ7K7WW).
-- Have questions or want to just say hi? Book a time during our office hours [on Calendly](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) or email us at solutions@fivetran.com.
\ No newline at end of file
+- Have questions or want to just say hi? Book a time during our office hours [on Calendly](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) or email us at solutions@fivetran.com. 
```

Comment:
We are trying to direct more folks to the Fivetran Community (dbt User Group) rather than directly to the office hours. 

Would you be able to update this section to instead look like the following (from our Zendesk README)
<img width="1076" alt="image" src="https://github.com/fivetran/dbt_twilio_source/assets/74217849/bfe5d285-a5ef-4a0e-ac3f-b9d3eba3cee4">

Reply:
updated